wagtail/rich_text/feature_registry.py

```diff
@@ -98,6 +103,18 @@ def get_converter_rule(self, converter_name, feature_name):
         except KeyError:
             return None
 
+    def register_frontend_rewriter(self, rewriter, order=300):
+        self.frontend_rewriters.append({"rewriter": rewriter, "order": order})
+
+    def get_frontend_rewriters(self):
+        if not self.has_scanned_for_features:
+            self._scan_for_features()
+
+        return [
+            r["rewriter"]
+            for r in sorted(self.frontend_rewriters, key=lambda e: e["order"])
```

Comment:
I think we’d need a test case for this ordering too.

Reply:
Added now, and good plan, as it picked up that I'd forgotten to do that